src/test/scala/uk.gov.nationalarchives.fileformat/RecordProcessorTest.scala

```diff
@@ -34,18 +34,18 @@ class RecordProcessorTest extends AnyFlatSpec with MockitoSugar with EitherValue
   "The processRecord method" should "send a message to the sqs queue" in {
```

Comment:
`siegfried` is mentioned in a couple of test cases. Might be worth renaming the tests to reflect that it's `droid` being used

The is a spelling mistake in one of the tests that might be worth correcting: Line 51 "reciept"